lib/util.js

```diff
@@ -72,6 +71,78 @@ const errorToString = Error.prototype.toString;
 var CIRCULAR_ERROR_MESSAGE;
 var Debug;
 
+// eslint-disable-next-line
+const escapeSequencesRegExp = /[\\'\x00-\x1f]/g;
+const keyStrRegExp = /^[a-zA-Z_][a-zA-Z_0-9]*$/;
+const textRegExp = /^[a-zA-Z_0-9 ,.?!+\-&"öäüÜÖÄß():;<>{}[\]=$]*$/;
+const colorRegExp = /\u001b\[\d\d?m/g;
+const numberRegExp = /^(0|[1-9][0-9]*)$/;
+
+// Table of character substitutions for text
+const escapeCodesText = {
+  39: "\\'",
+  92: '\\\\',
+  10: '\\n',
+  13: '\\r',
+  9: '\\t',
+  8: '\\b',
+  12: '\\f'
+};
+// Table of character substitutions for keys
+const escapeCodesKeys = {
+  39: "\\'",
+  92: '\\',
+  10: '\\n',
+  13: '\\r',
+  9: '\\t',
+  8: '\\b',
+  12: '\\f'
+};
+
+// Escape control characters and the backslash.
+// This is similar to JSON stringify escaping.
```

Comment:
You should at least mention the V8 version in here because the magic values used within could be tied to specific versions.

Reply:
I am not sure what magic values you mean. We define what characters we want to escape.

Reply:
Immediately right at the top of the function there is a comment that reads:

```js
// Some magic numbers that worked out fine while benchmarking
```

Reply:
Right, I forgot my comment